src/biogeophys/SoilHydrologyMod.F90

```diff
@@ -341,12 +341,12 @@ subroutine Infiltration(bounds, num_hydrologyc, filter_hydrologyc, num_urbanc, f
              endif
 
              !1. partition surface inputs between soil and h2osfc
-             qflx_in_soil(c) = (1._r8 - frac_h2osfc(c)) * (qflx_top_soil(c)  - qflx_sat_surf(c))
-             qflx_in_h2osfc(c) = frac_h2osfc(c) * (qflx_top_soil(c)  - qflx_sat_surf(c))          
+             qflx_in_soil(c)   = qflx_top_soil(c)  - qflx_sat_surf(c)
+             qflx_in_h2osfc(c) = qflx_top_soil(c)  - qflx_sat_surf(c)          
```

Comment:
**Comment from @martynpclark Aug 17, 2017**

Code modified because forcing is constant over the grid cell.